docs/add-your-app.md

```diff
@@ -0,0 +1,138 @@
+# Adding Your Apps
+
+
+## Automatic discovery
+While we would eventually like to automatically discover your application topology, 
+at this stage adding support for a new application is a manual step.
+The main challenge in automatic discovery stems from the fact that both kubernetes deployment descriptors is focused 
+on deployment instructions and lacks metadata required by ocopea in order to create application copies.
+
+## App templates 
+In the Ocopea terminology, every application topology is described by an "App template" entity.  
+In order to create a new app template users should use the "create-app-template" command of the hub-webapp 
+Ocopea service. See [invoking Commands](invoking-ocopea-commands.md)
+
+## Create App Template Command
+
+The "create-app-template" command expects the user to provide full description of the application topology as described
+in the 
+[hub-web swagger API](https://github.com/ocopea/orcs/blob/master/hub/webapp/web-api/src/main/resources/swagger.yaml).
+
+We'll use the lets-chat app template to understand the building block
```

Comment:
Would be nice if you give a link to the letschat github and/or the docker image/hub link. A lot of info comes from the docker file. It will make it easy for people to understand.